### PR TITLE
feat(invoices): update contents for spanish and english languages

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Update all contents for spanish and english languages about Mariano's feedback.

**Changes:**

- Updated all contents. For it we used the document:  https://docs.google.com/document/d/1T5tbzmi-MhaJwmcWDfIlnPnD8Cgppc1vBEI7bXAqSIw/edit

**Task:** [DAT-166](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-166)

### CODE REVIEW

**Comment by andresmoschini** [comment]
CC: @mrunge86 (in case you want to take a look...)

**Comment by andresmoschini** [comment]
👍

**Comment by fgchaio** [comment]
👏

File: src/i18n/en.js [hunk 1]
```diff
@@ -159,12 +159,12 @@ You'll find an Email with steps to follow.`,
...
-    error_msg: `There was an error gathering the invoices.`,
-    no_data_msg: `There are no invoices to show`,
-    no_download_msg: `There is no invoice to download`,
+    error_msg: `Ouch! An error occurred while trying to upload your invoices.`,
+    no_data_msg: `There are no issued invoices to show yet.`,
+    no_download_msg: `There is no invoice to download yet.`,
     paid_to_date_column: `Paid`,
     product_column: `Product`,
-    sub_title: `List of issued invoices`,
+    sub_title: `Here is the detail of the invoices we have issued. You may download each one.`,
```
Comment (cbernat) [inline src/i18n/en.js:167]: How about this?
You may download each one. --> Remember you can download each one of them individually.

Sounds more natural to me.

File: src/i18n/es.js [hunk 2]
```diff
@@ -158,12 +158,12 @@ Encontrarás un Email con los pasos a seguir.`,
...
-    error_msg: `Hubo un error al recopilar las facturas.`,
-    no_data_msg: `No hay facturas para mostrar`,
-    no_download_msg: `No hay factura para descargar`,
+    error_msg: `¡Ouch! Ocurrió un error al intentar cargar tus facturas.`,
+    no_data_msg: `Aún no hay facturas emitidas para mostrar.`,
+    no_download_msg: `Aún no hay factura para descargar`,
     paid_to_date_column: `Pagado`,
     product_column: `Producto`,
-    sub_title: `Listado de facturas emitidas`,
+    sub_title: `Aquí está detalle de las facturas que hemos emitido. Recuerda que puedes descargar cada una.`,
```
Comment (cbernat) [inline src/i18n/es.js:166]: How about this? If I'm grasping the idea ... 
Recuerda que puedes descargar cada una. --> Recuerda que puedes descargar cada una de ellas por separado.

**Pull request merged**